Source/Scene/BillboardCollection.js

```diff
@@ -1465,7 +1465,8 @@ define([
                 this._rsTranslucent = RenderState.fromCache({
                     depthTest : {
                         enabled : true,
-                        func : WebGLConstants.LEQUAL  // Allows label glyphs and billboards to overlap.
+                        // Allows label glyphs and billboards to overlap.
+                        func : ((this._blendOption === BlendOption.TRANSLUCENT) ? WebGLConstants.LEQUAL : WebGLConstants.LESS)
```

Comment:
What does `this._blendOption === BlendOption.TRANSLUCENT` really mean?  Does it mean "this billboard collection is for a label collection with backgrounds?"  If so, can we just asked the label collection or have the label collection pass in a private flag so this code is more obvious?